ssl/ssl_conf.c

```diff
@@ -680,18 +680,13 @@ static int cmd_RecordPadding(SSL_CONF_CTX *cctx, const char *value)
     if (commap == NULL)
         hs_padding = block_padding;
     OPENSSL_free(copy);
-    /*
-     * All we care about are non-negative values,
-     * the setters check the range
```

Comment:
So this came about when we replaced `atoi()` with `OPENSSL_strtoul()` (which can't return non-negative numbers). Since there is no range checking here, it might be helpful for future devs if we keep the part of the comment that the setters will check the range.